_episodes/01-intro.md

```diff
@@ -52,14 +52,14 @@ This means that, for example:
  - `weight` and `Weight` are different variables
 
 ## Types of data
-Python knows various types of data. Three common ones are:
+Python knows and can automatically recognize various types of data. Three common ones are:
 
 * integer numbers
 * floating point numbers, and
 * strings.
 
 In the example above, variable `weight_kg` has an integer value of `60`.
-To create a variable with a floating point value, we can execute:
+To assign that variable a floating point value, we can execute:
```

Comment:
I'm not sure about this change. Indeed, we're not creating a new variable here but assigning a new value to it. Personally, however, I'm OK with this inconsistency so that we don't use all the new terminology at once (create a variable _vs_ assign a {new} value). What do you think, @annefou?

Btw, @annefou is working on an update to this episode in which we'll be using different variable names (age, mass, name), which should make things clearer.